app/models/recipe.rb

```diff
@@ -0,0 +1,5 @@
+class Recipe < ApplicationRecord
```

Comment:
As you can see from the ER diagram, the recipe and food are connected to each other through the recipe_food. Kindly do so as setting up the associations between models in a correct way is a part of the core requirements. ⬇️
![image](https://user-images.githubusercontent.com/33268927/209437872-c593fb7a-966f-40cc-98c6-9465f32ffd7c.png)